sphinx_gallery/tests/test_gen_rst.py

```diff
@@ -8,8 +8,10 @@
 import tempfile
 
 import sphinx_gallery.gen_rst as sg
+import sphinx_gallery.notebook as gnb
```

Comment:
In general, those shorthands make the code harder to read. In my world, GNB means "Gaussian Naive Bayes" :).

Maybe you could simply use "from sphinx_gallery import notebook".